Description: Make correct PPD cache entry, so that cacched PPD entries from *.drv files get recognized.
Bug: http://www.cups.org/str.php?L3921
Author: Till Kamppeter

--- a/scheduler/cups-driverd.cxx
+++ b/scheduler/cups-driverd.cxx
@@ -1401,9 +1401,7 @@
   * Add a dummy entry for the file...
   */
 
-  httpAssembleURIf(HTTP_URI_CODING_ALL, uri, sizeof(uri), "drv", "", "", 0,
-		   "/%s", name);
-  add_ppd(name, uri, "", "", "", "", "", "", mtime, size, 0,
+  add_ppd(name, name, "", "", "", "", "", "", mtime, size, 0,
           PPD_TYPE_DRV, "drv");
   ChangedPPD = 1;
 
